lib/cargo.js

```diff
@@ -105,6 +106,18 @@ export function provideBuilder() {
         }
       }
 
+      // Checks if a file pointed by a message relates to the Rust source code and
+      // corrects it if needed and if possible
+      function normalizePath(path) {
+        if (rustSrcPath && path.length > 6) {
+          const prefix = path.substring(0, 7);
```

Comment:
It's the length of the marker. I agree, they might be confusing here, I should use a named constant instead. I'll fix it.